Tests/ApolloCodegenTests/CodeGeneration/Templates/SelectionSet/SelectionSetTemplateTests.swift

```diff
@@ -6952,6 +7003,53 @@ class SelectionSetTemplateTests: XCTestCase {
       .to(equalLineByLine(predator_expected, atLine: 21, ignoringExtraLines: true))
   }
 
+  func test__render_nestedSelectionSet__givenInlineFragmentWithOnlyReservedField_doesNotRenderCompositeInlineFragment() async throws {
```

Comment:
Can we add the link to the issue here as well?
```suggestion
// Related to https://github.com/apollographql/apollo-ios/issues/3326
  func test__render_nestedSelectionSet__givenInlineFragmentWithOnlyReservedField_doesNotRenderAsCompositeInlineFragment() async throws {
```